docs/source/standard.md

```diff
@@ -255,39 +255,41 @@ The ability of the organisation to support users who want to browse the data ava
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+| Statement                                                                                                                                   | Guidance                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| ------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Regular vulnerability scans of TRE infrastructure should be conducted                                                                       | Ensuring that scans are done on a regular basis can enable TRE operators can identify and address weaknesses that may have been introduced during the TRE's operational lifetime. One way this can be achived is by ensuring that all machines (whether physical or virtual/cloud-based) are set to check for operating system updates on a regular basis, and install any security patches pushed by the developers of those operating systems. |
+| TREs should undergo compliance checking of machine and resource configurations to ensure adherence to security standards and best practices | Adhering to the highest level of security standards suggested by the IT department of the TRE operators organisation will reduce the risk of security breaches and data leakage. This should be done in advance of any sensitive data being uploaded to the TRE.                                                                                                                                                                                 |
+| TREs should undergo specific compliance checks required by law                                                                              | Many industry regulations and standards require organisations to perform security tests and other compliance checks, for example GDPR. By conducting these tests, organisations can demonstrate due diligence, increasing trust in the TRE and avoiding penalties for the TRE operators.                                                                                                                                                         |
```

Comment:
What checks are required by GDPR?

Reply:
Maybe namedropping GDPR is more specific than we want, but [apparently](https://www.itgovernance.co.uk/green-papers/penetration-testing-and-the-gdpr):

> Article 32 of the GDPR requires organisations to regularly test and evaluate the effectiveness of the technical and organisational measures employed to protect personal data, and penetration testing is an effective way of assessing your technical defences.

Reply:
Removed